server/conf/timely-standalone.yml

```diff
@@ -0,0 +1,47 @@
+timely:
+  ip: 0.0.0.0
+  port:
+    put: 54321
+    query: 54322
+    websocket: 54323
+  instance-name: TimelyStandalone
```

Comment:
The listen section seems clunky. suggest putting http-port under http as port, etc. 

Conceivably the http and ws could be bound to different IPs. (scope creep)

Finally, I think we dropped the http.redirect-path as a thing, but it probably just wasn't removed from the config.